css/style.css

```diff
@@ -0,0 +1,868 @@
+  /* background-repeat: no-repeat;
+  background-position: center top; */
+  width: 22px;
+  height: 22px;
+  margin-right: 7px;
+}
+
+.order-title {
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  margin-right: 40px;
+  text-shadow: 0 3px 4px rgba(0, 0, 0, 0.3);
+}
+
+.btn {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  color: #e51937;
+  background: #fff;
+  font-family: 'Montserrat';
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  transition: background-color 0.3s ease-out;
+}
+
+.btn:hover {
+  background-color: #e51937;
+  color: #fff;
+}
+
+.intro-arrow {
+  text-align: center;
+}
+
+.about__wrapper {
+  height: 900px;
+  position: relative;
+}
+
+.about-bg__img {
+  background: url(../img/img2.svg) center/cover no-repeat;
+  height: 900px;
+  max-width: 100%;
+}
+
+.about-title {
+  position: absolute;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  color: #E51937;
+  left: 80px;
+  top: 0;
+  margin: 30px 0;
+  width: 300px;
+}
+
+.box {
+  width: 440px;
+  height: 900px;
+  top: 0;
+  right: 0;
+  background: #e51937;
+  color: #fff;
+  position: absolute;
+}
+
+.box-right__inner {
+  width: 320px;
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.title {
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  margin-bottom: 12px;
+}
+
+.text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-bottom: 55px;
+}
+
+.about-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 267px;
+  margin-bottom: 171px;
+}
+
+.about__feature {
+  display: flex;
+  align-items: center;
+}
+
+.icon04 {
+  background: url(../img/icon04.svg) center no-repeat;
+  /* background-repeat: no-repeat;
+  background-position: center; */
+  margin-right: 24px;
+  width: 36px;
+  height: 53px;
+}
+
+.icon05 {
+  background: url(../img/icon05.svg) center/contain no-repeat;
+  /* background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain; */
+  margin-right: 20px;
+  width: 55px;
+  height: 34px;
+}
+
+.icon06 {
+  background: url(../img/icon06.svg) center no-repeat;
+  margin-right: 24px;
+  width: 36px;
+  height: 52px;
+}
+
+.order {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  left: 50px;
+  bottom: 60px;
+}
+
+.info-icon {
+  background: url(../img/info-icon.svg) center top no-repeat;
+  margin-right: 10px;
+  width: 22px;
+  height: 22px;
+}
+
+.info {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  margin-right: 28px;
+}
+
+.info:hover {
+  font-size: 14px;
+  margin-top: -3px;
+  margin-right: 18px;
+}
+
+.btn-white {
+  border-color: #fff;
+  color: #fff;
+  background: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5948710)